offline/cable_LUC_EXPT.F90

```diff
@@ -246,6 +246,9 @@ SUBROUTINE LUC_EXPT_INIT(LUC_EXPT)
             start=(/1,1,LUC_EXPT%CTSTEP/),count=(/xds,yds,1/) )
        CALL HANDLE_ERR(STATUS, "Reading from "//LUC_EXPT%TransFile(i) )
        DO k = 1, mland
+          WRITE(*,*) "land kxy   ",k, land_x(k), land_y(k)
```

Comment:
don;t like this - adding yet more debugging outputs to code - should be stripped out prior to merge.

Reply:
If it's not necessary, take it out now.

Reply:
I've opend a new issue for this [Issue 333](https://github.com/CABLE-LSM/CABLE/issues/333)